python/src/datafusion.rs

```diff
@@ -175,6 +175,10 @@ mod tests {
     }
 
     impl LazyBatchGenerator for TestBatchGenerator {
+        fn as_any(&self) -> &dyn Any {
```

Comment:
due to https://datafusion.apache.org/library-user-guide/upgrading.html#add-as-any-method-to-lazybatchgenerator